variables.tf

```diff
@@ -173,6 +173,12 @@ variable "get_weka_io_token_secret_id" {
   default     = ""
 }
 
+variable "weka_admin_password_secret_id" {
```

Comment:
why is it needed?

Reply:
we needed it for get_weka_io since until now we passed it as TF var, here it isn't required since we generate the password